cloud/client.go

```diff
@@ -181,6 +182,13 @@ func (c *client) doCall(ctx context.Context, method, url string, opts ...request
 	for attempt := 0; attempt < maxAttempt; attempt++ {
 		status, body, err = c.doCallImp(ctx, r, method, url, opts...)
 
+		if err != nil && strings.Contains(err.Error(), "context canceled") {
```

Comment:
nice fix! I've seen this crop up a couple of times which resulted in me having to hit ctrl-c a second time.